src/design-tokens/sizes.yml

```diff
@@ -100,3 +100,15 @@ props:
     value: 3
     type: modular/em
     category: rhythm
+  - name: spacing_horizontal_condensed
+    value: -1
+    type: modular/em
+    category: spacing
+  - name: spacing_horizontal
+    value: 1
+    type: modular/em
+    category: spacing
+  - name: spacing_horizontal_generous
+    value: 3
+    type: modular/em
+    category: spacing
```

Comment:
Perhaps we could call these `gap_` instead of `spacing_` to be more concise?

Can we add a `comment` to provide more clarity?